jobs/service-fabrik-broker/templates/config/keepalived/master.sh.erb

```diff
@@ -0,0 +1,27 @@
+iaas_name=<%= p('backup.provider.name') %>
+if [[ "$iaas_name" == "aws" ]]; then
+
+  SUBNET_ID='<%= spec.networks.sf_service_fabrik_broker.cloud_properties.subnet %>'
+  SOURCE_IP=<%= spec.ip %>
+  <% internal = p('internal') %>
+
+  BACKUP_OBJ='<%= JSON.dump(p('backup.provider', nil)) %>'
+  AWS_CLI_CMD=/var/vcap/packages/awscli/bin/aws
+  JQ_CMD=/var/vcap/packages/jq/bin/jq
+
+  ACCESS_KEY_ID="$(echo "${BACKUP_OBJ}" | "${JQ_CMD}" -r '.keyId')"
```

Comment:
also after the aws credential policy hardening , not sure the backup user will have access to perform required IAAS operations for HA.

Reply:
You are right, for broker user its ; 

```
 {
            "Action": "ec2:*",
            "Effect": "Allow",
            "Resource": "*"
        },
        {
            "Action": "s3:*",
            "Effect": "Allow",
            "Resource": "*"
        }
```